fitbenchmarking/core/fitting_benchmarking.py

```diff
@@ -625,14 +624,15 @@ def perform_fit(controller, options, grabbed_output):
             raise ControllerAttributeError(
                 "Either the computed runtime or chi_sq values "
                 "was a NaN.")
+    except ValidationException as ex:
+        LOGGER.warning(str(ex))
+        controller.flag = 7
     except Exception as ex:  # pylint: disable=broad-except
         LOGGER.warning(str(ex))
 
         # Note: Handle all exceptions as general exception to cover case
         #       where software re-raises our exception as a new type.
-        error_flags = {MaxRuntimeError: 6,
-                       IncompatibleJacobianError: 7,
-                       IncompatibleHessianError: 7}
```

Comment:
Just want to check that the validation still works described in here https://github.com/fitbenchmarking/fitbenchmarking/pull/937 

because I had a feeling the validation exception was being caught and re-raised as a different error?

Reply:
I think this should raise any validation exception before ~software~ *bumps* gets to run, so it won't be re-raised as something else.